PWGDQ/TableProducer/tableMakerMC.cxx

```diff
@@ -244,6 +244,9 @@ struct TableMakerMC {
     uint64_t trackFilteringTag = 0;
     uint8_t trackTempFilterMap = 0;
     for (auto& collision : collisions) {
+      if (!collision.has_mcCollision()) {
```

Comment:
As we discussed, please add a //TODO:  statement here, so we can comeback to this in the future